tools/frb_internal/lib/src/makefile_dart/generate.dart

```diff
@@ -371,18 +375,25 @@ Future<void> generateRunFrbCodegenCommandIntegrate(
   });
 }
 
-Future<RunCommandOutput> executeFrbCodegen(
-  String cmd, {
-  required String relativePwd,
-  required bool coverage,
-  bool postRelease = false,
-  required String coverageName,
-  bool nightly = false,
-}) async {
+Future<RunCommandOutput> executeFrbCodegen(String cmd,
+    {required String relativePwd,
+    required bool coverage,
+    bool postRelease = false,
+    required String coverageName,
+    bool nightly = false,
+    List<String> features = const []}) async {
   if (postRelease) {
     assert(!coverage);
-    return await exec('flutter_rust_bridge_codegen $cmd',
-        relativePwd: relativePwd);
+    if (cmd == "generate") {
+      String f = features.map((f) {
+        return '--features $f';
+      }).join(' ');
+      return await exec('flutter_rust_bridge_codegen $cmd $f',
+          relativePwd: relativePwd);
+    } else {
+      return await exec('flutter_rust_bridge_codegen $cmd',
+          relativePwd: relativePwd);
+    }
```

Comment:
maybe we can keep this function unchanged. Instead, when users want to add features, they can provide `cmd` argument with corresponding extra string

Reply:
(this is not handled)